src/NHibernate/Engine/CollectionEntry.cs

```diff
@@ -302,9 +302,20 @@ public void PostInitialize(IPersistentCollection collection)
 		{
```

Comment:
The single argument `PostInitialize` should be obsoleted. Or is there any reason to keep it, beside avoiding a breaking change? It seems now only called from the two arguments `PostInitialize`.

Reply:
Done